team_list.html

```diff
@@ -46,10 +46,10 @@
       <div class="right">
         <ul>
           <li class="about"><a  href="index.html#about">About</a></li>
-          <li class="services"><a target="_blank" href="https://github.com/orgs/OpenCodeEra/repositories">Projects</a></li>
+          <li class="services"><a href="index.html#projects">Projects</a></li>
           <li><a href="/team_list.html">Team</a></li>
           <li class="contact-us"><a  href="index.html#contact">Contact Us</a></li>
-          <li class="about"><a  href="index.html">FAQ</a></li>
+          <li class="about"><a  href="index.html#faq">FAQ</a></li>
```

Comment:
index.html#faq i.e. line 52 is not working !

Reply:
```suggestion
          <li class="about"><a  href="index.html#faq">FAQ</a></li>
          It is perfectly working in my local environment. I think you should recheck it.
```

Reply:
I have added the link to the #faq section present in <index.html> to the file <team_list.html>.

Reply:
Okay... It's working.
Push the code of `team_list.html` file only. So that I can merge it.

Reply:
I have pushed the code and revoked all the changes made to <index.html> file, also, updated <team_list.html> as required.